danspeech/DanSpeechRecognizer.py

```diff
@@ -39,10 +41,28 @@ def __init__(self, model_name=None, lm_name=None,
             self.lm = None
             self.decoder = None
 
+        # Streaming declarations
+        self.streaming = False
+        self.full_output = []
+        self.iterating_transcript = ""
+
+        self.second_model = DanSpeechPrimary()
+        self.audio_config = self.second_model.audio_conf
+        self.second_model =  self.second_model.to(self.device)
+        self.second_model.eval()
+
+        self.second_decoder = DSL3gram()
+
+        # When updating model, always update decoder because of labels
+        self.second_decoder = decoder = BeamCTCDecoder(labels=self.second_model.labels, lm_path=self.second_decoder,
```

Comment:
I think we should add an exception here for when BeamCTC isn't installed, I'm guessing that will happen rather often.

Reply:
Agree, it will be refactored in upcoming PR! :)